src/grid/grid3D.cpp

```diff
@@ -136,7 +140,8 @@ void Grid3D::Initialize(struct parameters *P)
   C_cfl = 0.3;
   
   #ifdef AVERAGE_SLOW_CELLS
-  H.min_dt_slow = 1e-100; //Initialize the minumum dt to a tiny number
+  // H.min_dt_slow = 1e-10; //Initialize the minumum dt to a tiny number
+  H.min_dt_slow = 1e-2;
```

Comment:
should this be wrapped in ifdef DUST as well? not really sure how changing min_dt_slow affects other parts of code